### PR TITLE
Homework 02.12.2021

### PR DESCRIPTION
**Pull request by nickbelous**
- realize 5 testcases for our app

### CODE REVIEW

File: homework_02_12.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        # Found the text
+        checked_text_tc1 = driver.find_element(by=By.XPATH, value=".//H4")
+        # Click on the link
+        checked_text_tc1.click()
+        sleep(2)
+        # Check link on the page
+        checked_text_tc1 = driver.find_element(by=By.XPATH, value=".//H4")
+        checked_text_tc1.is_displayed()
+
+    """Check text "Remember Writing?" is display in main page """
```
Comment (QALightLecorPython) [inline homework_02_12.py]: Docstring should be under the method it describing, not in-between

File: homework_02_12.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+"""Home work - practice with XPATH"""
```
Comment (QALightLecorPython) [inline homework_02_12.py]: Lets not split files on home/class work, but store tests based on related pages e.g `test_start_page`

File: homework_02_12.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+from time import sleep
+
+from selenium.webdriver.chrome import webdriver
+from selenium.webdriver.common.by import By
+
+
+class TestHomeWorkTestXpath:
```
Comment (QALightLecorPython) [inline homework_02_12.py]: Lets create better name for the class

File: homework_02_12.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    """TC1 - After click to text "Complex app for testing - QA" redirected to main page """
+
+    def test_test_case1(self):
```
Comment (QALightLecorPython) [inline homework_02_12.py]: Lest call tests by functionality instead of `test_case1`

**Review by QALightLecorPython — CHANGES_REQUESTED** [review]
Generally looks good, but some style things have to be updated

**Comment by nickbelous** [comment]
no reason to create additional file to homework

**Pull request closed without merging**